commcare-cloud/commcare_cloud.py

```diff
@@ -0,0 +1,125 @@
+# coding=utf-8
```

Comment:
`#!/usr/bin/env python`?

Reply:
I made this executable through the `entry_points.console_scripts` setting in setup.py here: https://github.com/dimagi/commcarehq-ansible/pull/1032/files#diff-1028db3bff7bbf8d651eee1350b2c8f9R10. So I don't actually need to make this file itself executable.

Reply:
Ah! That's cool. I didn't know that was a thing.